src/main.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import './styles/droneList.css'
+
+const drones = [
+    {
+        id: 0,
+        status: "parked"
+    },
+    {
+        id: 1,
+        status: "delivering"
+    },
+    {
+        id: 2,
+        status: "returning"
+    },
+    {
+        id: 3,
+        status: "undefined"
+    }
+];
+
+function Main() {
+    let list = [];
+    for(let i = 0; i < drones.length; i++) {
+        if(drones[i].status === "parked") {
+            list[i] = <li className="green">
+                {drones[i].id} - {drones[i].status};
+            </li>
+        }
+        else if(drones[i].status === "delivering") {
```

Comment:
I would modify the following code:
create a switch block function and based on the drones status return the correct css class name.
then I would create the list like this 
```
const dronesList = drones.map((drone, index) => (
    <li key={index} className={getClassName(drone.status)}>
        {drone.id} - {drone.status}
    </li>
```

Reply:
Done